components/graphs/Graph.tsx

```diff
@@ -1,11 +1,11 @@
 import { LegendLineGraphIcon } from '@openclimatefix/nowcasting-ui.icons.icons';
 import {
   forecastDataOverDateRange,
-  formatter,
   getCurrentTimeForecastIndex,
+  weekdayFormatter,
 } from 'lib/graphs';
 import { useSiteData } from 'lib/hooks';
-import { FC, useState } from 'react';
+import { FC, useState, memo } from 'react';
```

Comment:
Unused import
```suggestion
import { FC, useState } from 'react';
```

Reply:
oops sorry i was trying out some recharts custom tick label stuff that needed that